packit/upstream.py

```diff
@@ -376,9 +380,16 @@ def local_project(self) -> Optional[LocalProject]:
     @property
     def working_dir(self) -> Optional[Path]:
         if not self._working_dir:
-            self._working_dir = Path(tempfile.mkdtemp())
+            if self.handler_kls == SandcastleCommandHandler:
+                path = (
+                    Path(self.config.command_handler_work_dir) / "non-git-working-dir"
```

Comment:
Wouldn't it be better to run `pull-from-upstream` that doesn't have upstream git from the dist-git repo? :thonk:

Reply:
yeah that would require even more invasive changes, and this way people can run some actions in this directory